pkg/storkctlcli/storkctlcli.go

```diff
@@ -0,0 +1,161 @@
+package storkctlcli
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	storkv1 "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	"github.com/libopenstorage/stork/pkg/storkctl"
+	"github.com/portworx/sched-ops/task"
+	"github.com/sirupsen/logrus"
+
+	"github.com/portworx/torpedo/pkg/aetosutil"
+	"github.com/portworx/torpedo/pkg/log"
+)
+
+var dash *aetosutil.Dashboard
+
+const (
+	drPrefix = "automation-"
+	actionRetryTimeout = 10 * time.Minute
+	actionRetryInterval = 10 * time.Second
+)
+
+var (
+	migSchedNs = "kube-system"
+)
+
+func ScheduleStorkctlMigrationSched(schedName, clusterPair, namespace string, extraArgs map[string]string) error {
+	if namespace != "" {
+		migSchedNs = namespace 
```

Comment:
Add a new param to the function that accepts the namespace, if it's empty then default to `migSchedNs` that way one can use this function to create clusterpair in any namespace.

Reply:
Sorry, I didnt get you, It has a namespace argument and its checking if someone doesnt pass a namespace then it that case it will use kube-system. Defined migSchedNs as a global param above. Please correct me if I miss anything here -

var (
	migSchedNs = "kube-system"
)

func ScheduleStorkctlMigrationSched(schedName, clusterPair, namespace string, extraArgs map[string]string) error {
	if namespace != "" {
		migSchedNs = namespace 
	}